sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSender.java

```diff
@@ -289,6 +295,86 @@ public void dispose() {
         tokenManager.close();
     }
 
+    @Override
```

Comment:
There is a lot of duplicated code here. You should be able to condense the common parts and not.